src/pages/articles/Articles.js

```diff
@@ -0,0 +1,34 @@
+import React, { useContext } from "react";
+import Body from "../../components/Body";
+
+import { PageContext } from "../../PageContextProvider";
+
+function Articles() {
+  const { articles } = useContext(PageContext);
+  console.log("POL", articles);
+  return (
```

Comment:
We need a loading state here to conditionally render a loading message based on whether the fetch request for articles has been completed or not